renderer_elements.js

```diff
@@ -112,18 +147,6 @@ class State extends SDFGElement {
             ctx.strokeRect(clamped.x, clamped.y, clamped.w, clamped.h);
         }
 
-        // If collapsed, draw a "+" sign in the middle
-        if (this.data.state.attributes.is_collapsed) {
-            ctx.beginPath();
-            ctx.moveTo(this.x, this.y - LINEHEIGHT);
-            ctx.lineTo(this.x, this.y + LINEHEIGHT);
-            ctx.stroke();
-            ctx.beginPath();
-            ctx.moveTo(this.x - LINEHEIGHT, this.y);
-            ctx.lineTo(this.x + LINEHEIGHT, this.y);
-            ctx.stroke();
-        }
-
```

Comment:
why remove this?

Reply:
This is an embarrassing mistake, thanks for catching it! This hunk was reverted.